lib/action_kit/client.rb

```diff
@@ -0,0 +1,17 @@
+module ActionKit
```

Comment:
So it is: https://github.com/EricBoersma/actionkit_connector/blob/master/lib/actionkit_connector.rb#L33